Doc/data/refcounts.dat

```diff
@@ -2863,6 +3432,61 @@ PyUnicodeDecodeError_SetStart:int:::
 PyUnicodeDecodeError_SetStart:PyObject*:exc:0:
 PyUnicodeDecodeError_SetStart:Py_ssize_t:start::
 
+PyUnicodeEncodeError_GetEncoding:PyObject *::+1:
+PyUnicodeEncodeError_GetEncoding:PyObject *:exc:0:
+
+PyUnicodeEncodeError_GetEnd:int:::
+PyUnicodeEncodeError_GetEnd:PyObject *:exc:0:
+PyUnicodeEncodeError_GetEnd:Py_ssize_t *:end::
+
+PyUnicodeEncodeError_GetObject:PyObject *::+1:
+PyUnicodeEncodeError_GetObject:PyObject *:exc:0:
+
+PyUnicodeEncodeError_GetReason:PyObject *::+1:
+PyUnicodeEncodeError_GetReason:PyObject *:exc:0:
+
+PyUnicodeEncodeError_GetStart:int:::
+PyUnicodeEncodeError_GetStart:PyObject *:exc:0:
+PyUnicodeEncodeError_GetStart:Py_ssize_t *:start::
+
+PyUnicodeEncodeError_SetEnd:int:::
+PyUnicodeEncodeError_SetEnd:PyObject *:exc:0:
+PyUnicodeEncodeError_SetEnd:Py_ssize_t:end::
+
+PyUnicodeEncodeError_SetReason:int:::
+PyUnicodeEncodeError_SetReason:PyObject *:exc:0:
+PyUnicodeEncodeError_SetReason:const char *:reason::
+
+PyUnicodeEncodeError_SetStart:int:::
+PyUnicodeEncodeError_SetStart:PyObject *:exc:0:
+PyUnicodeEncodeError_SetStart:Py_ssize_t:start::
+
+PyUnicodeTranslateError_GetEnd:int:::
+PyUnicodeTranslateError_GetEnd:PyObject *:exc:0:
+PyUnicodeTranslateError_GetEnd:Py_ssize_t *:end::
+
+PyUnicodeTranslateError_GetObject:PyObject *::+1:
+PyUnicodeTranslateError_GetObject:PyObject *:exc:0:
+
+PyUnicodeTranslateError_GetReason:PyObject *::+1:
+PyUnicodeTranslateError_GetReason:PyObject *:exc:0:
+
+PyUnicodeTranslateError_GetStart:int:::
+PyUnicodeTranslateError_GetStart:PyObject *:exc:0:
+PyUnicodeTranslateError_GetStart:Py_ssize_t *:start::
+
+PyUnicodeTranslateError_SetEnd:int:::
+PyUnicodeTranslateError_SetEnd:PyObject *:exc:0:
+PyUnicodeTranslateError_SetEnd:Py_ssize_t:end::
+
+PyUnicodeTranslateError_SetReason:int:::
+PyUnicodeTranslateError_SetReason:PyObject *:exc:0:
+PyUnicodeTranslateError_SetReason:const char *:reason::
+
+PyUnicodeTranslateError_SetStart:int:::
+PyUnicodeTranslateError_SetStart:PyObject *:exc:0:
+PyUnicodeTranslateError_SetStart:Py_ssize_t:start::
+
 PyWeakref_Check:int:::
 PyWeakref_Check:PyObject*:ob::
```

Comment:
This, as well as `PyWeakref_CheckRef` and `PyWeakref_CheckProxy` below it, need reference count effects.
```suggestion
PyWeakref_Check:PyObject*:ob:0:
```

Reply:
Addressed in https://github.com/python/cpython/pull/127451.